presto-docs/src/main/sphinx/overview/concepts.rst

```diff
@@ -151,11 +151,11 @@ Statement
 ^^^^^^^^^
 
 Presto executes ANSI-compatible SQL statements.  When Presto
-documentation refers to a statement we are refering to statements as
+documentation refers to a statement we are referring to statements as
```

Comment:
Thanks for finding and fixing these typos. Can you move this to a separate commit, since it's unrelated to the Hive change?